EasyPost/Billing.cs

```diff
@@ -0,0 +1,93 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using RestSharp;
+
+namespace EasyPost
+{
+    public static class Billing
+    {
+        /// <summary>
+        ///     Delete a payment method.
+        /// </summary>
+        /// <param name="paymentMethodId">ID of the payment method to delete.</param>
+        /// <returns>Whether the request was successful or not.</returns>
+        public static async Task<bool> DeletePaymentMethod(string paymentMethodId)
+        {
+            Request request = new Request($"credit_cards/{paymentMethodId}", Method.Delete);
+
+            return await request.Execute();
+        }
+
+        /// <summary>
+        ///     Fund your wallet from a specific payment method.
+        /// </summary>
+        /// <param name="amount">Amount to fund.</param>
+        /// <param name="primaryOrSecondary">Which type of payment method to use to fund.</param>
+        /// <returns>True if successful, false otherwise.</returns>
+        public static async Task<bool> FundWallet(string amount, PaymentMethod.Priority primaryOrSecondary)
+        {
+            PaymentMethodObject paymentMethod = await GetPaymentMethodByPriority(primaryOrSecondary);
+
+            Request request = new Request($"credit_cards/{paymentMethod.id}/charges", Method.Post);
```

Comment:
Issue: The whole point of consolidating these is to do a conditional check on what payment method was passed in. If a user passes in a BankAccount ID here, this will fail because it needs to be routed to the `bank_accounts` endpoint instead.

We should check what kind of ID is passed and change the URL accordingly

Reply:
Resolved.